Widescreen patch
Twin Castle, Emille's painting prison
Lisa
Old: Looked in Emille's current (hidden) position to face mirror.
New: Replace Emille reference with static position.
--- original
+++ modified
@@ -373,10 +373,11 @@
 gosub inl[:LABEL_16]
 mov 0x1e, stor[24]
 wait stor[24]
-mov 0x1, stor[24]
-mov 0xb, stor[25]
-mov 0x1, stor[26]
-gosub inl[:LABEL_42]
+mov 0xfffffe54, stor[24]
+mov 0x167, stor[25]
+mov 0xb, stor[26]
+mov 0x1, stor[27]
+gosub inl[:LABEL_36]
 mov 0x8, stor[24]
 mov 0x1, stor[25]
 gosub inl[:LABEL_16]
